docs/advanced_topics/embeds.rst

```diff
@@ -191,6 +191,40 @@ For example, this is how you can instruct Youtube to return videos in HTTPS
     Wagtail will not try to run any other finder, even if the chosen one didn't
     return an embed.
 
+.. _Instagram:
+
+Facebook and Instagram
+----------------------
+
+As of October 2020, Facebook deprecated their public oEmbed APIs. If you would
+like to embed Facebook or Instagram posts in yoru site, you will need to
+use the new authenticated APIs. This requires you to set up a Facebook
+Developer Account and create a Facebook App that includes the oEmbed Product.
+Instructions for creating the neccessary app are in requirements sections of the
+`Facebook <https://developers.facebook.com/docs/plugins/oembed>`_
+and `Intagram <https://developers.facebook.com/docs/instagram/oembed>`_ documentation.
+
+Once you have your app access tokens (App ID and App Secret), add the Facebook and/or
+Instagram finders to your ``WAGTAILEMBEDS_FINDERS`` setting and configure them with
+the App ID and App Secret from your app:
+
+.. code-block:: python
+
+    WAGTAILEMBEDS_FINDERS = [
```

Comment:
I _love_ that you made these their own finders and added the config to the dicts down below. Wish I would have thought of that!